client/sites/settings/index.tsx

```diff
@@ -48,7 +48,7 @@ export default function () {
 		clientRender
 	);
 	page(
-		'/sites/settings/administration/:site/reset-site',
+		'/sites/settings/site/:site/reset-site',
```

Comment:
@fushar I'm considering changing the URLs from `/sites/settings/administration*` to `/sites/settings/site*`, as they are placed under the General sub-tab within the Settings tab. Please let me know if there's anything I should be aware of.

Reply:
No objection!